### PR TITLE
Add instructions for accessing ingress gateway installed using Helm

### PR DESCRIPTION
**Pull request by frankbu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 1]
```diff
@@ -466,7 +483,7 @@ Setting the ingress IP depends on the cluster provider:
 1.  _Other environments:_
 
     {{< text bash >}}
-    $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
+    $ export INGRESS_HOST=$(kubectl get po -l "${INGRESS_NAME}" -n  "${INGRESS_NS}" -o jsonpath='{.items[0].status.hostIP}')
```
Comment (ericvn) [inline content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md:486]: Does this need an `istio=` or maybe it was wrong originally.

Comment (frankbu) [reply content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md]: Thanks, good catch! My change here is wrong.

**Comment by ericvn** [comment]
LGTM - Will see if @luis-giraldo has a comment here or in https://github.com/istio/istio.io/pull/12263.

**Comment by luis-giraldo** [comment]
Yup, this is definitely a more general and clear approach.

**Review by ericvn — APPROVED** [review]

**Pull request merged**